BurnOutSharp/ProtectionType/Macrovision.cs

```diff
@@ -13,8 +15,23 @@ namespace BurnOutSharp.ProtectionType
     /// <summary>
     /// This is a placeholder for all Macrovision-based protections. See partial classes for more details
     /// </summary>
-    public partial class Macrovision : IPathCheck, IPortableExecutableCheck
+    public partial class Macrovision : IPathCheck, INewExecutableCheck, IPortableExecutableCheck
     {
+        /// <inheritdoc/>
+        public string CheckNewExecutable(string file, NewExecutable nex, bool includeDebug)
+        {
+            // Get the DOS stub from the executable, if possible
+            var stub = nex?.DOSStubHeader;
+            if (stub == null)
+                return null;
+
+            string cDilla = CDillaCheckNewExecutable(file, nex, includeDebug);
```

Comment:
**Question:** Were you planning on doing a list of detected protections similar to the PE checks?